### PR TITLE
Add new serializer empty_data feature doc

### PR DESCRIPTION
**Pull request by Nek-**
This is the documentation related to the following feature: https://github.com/symfony/symfony/pull/25493

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -913,6 +913,36 @@ These are the options available:
 ``remove_empty_tags``
     If set to true, removes all empty tags in the generated XML.
 
+Handling Value Objects
+----------------------
+
+Value Objets are difficult to handle because they often require parameters in the constructor. If the input omit one
+of theses parameters the serializer will throw an exeception because it can't create the object.
```
Comment (fabpot) [inline components/serializer.rst]: typo `exeception`

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by Nek-** [comment]
@fabpot fixed

**Review by javiereguiluz — APPROVED** [review]
@Nek- thanks for contributing this doc. I liked it a lot and everything is well explained. I just made a minor reword to focus more on the general use case ("Handling Constructor Arguments") instead of the "value object" detail. Thanks!

File: components/serializer.rst [hunk 2]
```diff
@@ -913,6 +913,40 @@ These are the options available:
 ``remove_empty_tags``
     If set to true, removes all empty tags in the generated XML.
 
+Handling Constructor Arguments
+------------------------------
+
+If the constructor of a class defines arguments, as usually happens with
```
Comment (dunglas) [inline components/serializer.rst]: It's not exact: the serializer deals well with constructor arguments, if they are present in the object to deserialize.

The `default_constructor_arguments` is useful only when the value has not been provided in the data to deserialize.

**Comment by Nek-** [comment]
Fixed and rebased. 👍

**Comment by javiereguiluz** [comment]
@Nek- I'm sorry it took us so long to merge. Thanks!

**Pull request merged**